ds_judgements_public_ui/templates/layouts/base.html

```diff
@@ -36,7 +36,28 @@
     <header class="page-header">
       {% block header %}
         <div class="page-header__flex-container container py-1">
-          {% include "includes/header_nav.html" with title=context.page_title link=request.path %}
+          <div class="page-header__nav">
+            <div class="page-header__site-logo">
+              <a href="{% url "home" %}" id="home-link">{% translate "home" %}</a>
+            </div>
+            <div class="page-header__breadcrumb">
+              <nav class="page-header__breadcrumb-flex-container"
+                   aria-label="Breadcrumb">
+                <ol>
+                  <li>
+                    <span class="page-header__breadcrumb-you-are-in">You are in:</span>
+                    {% if title == "home" %}
+                      {% translate "common.findcaselaw" %}
+                    {% else %}
+                      <a href="{% url 'home' %}">{% translate "common.findcaselaw" %}</a>
+                    {% endif %}
+                  </li>
+                  {% block breadcrumbs %}
```

Comment:
sadly have had to copy this out from header_nav.html.

Have left it as it was in judgment.html for now but once judgment.html and base.html have been deduplicated we will just have the one copy of this code.

It was a shame that I couldnt figure out a way to mix inheritance blocks and includes. I tried various combinations of files but it doesnt work.

At minimum, I understand that you can only ever extend from 1 parent templatye, so including another file which extends from another file will not work.

Any suggestions much appreciated 🙏

Reply:
Possibly this is more effort to fix than it is to just scrub `judgment.html` in favour of a better-behaving `base.html`.

Reply:
Possibly this is more effort to fix than it is to just scrub `judgment.html` in favour of a better-behaving `base.html`.

Reply:
yeah definitely need to fix up the base/judgment html duplication but it would still be nice to be able to abstract some of the base html, but I will leave for now